src/java/org/apache/cassandra/index/sai/StorageAttachedIndex.java

```diff
@@ -664,7 +712,7 @@ public void insertRow(Row row)
         @Override
         public void updateRow(Row oldRow, Row newRow)
         {
-            insertRow(newRow);
+            indexContext.getMemtableIndexManager().update(key, oldRow, newRow, memtable);
```

Comment:
This was using `insertRow`, which calls `adjustMemtableSize`. Don't we need that here too?

Reply:
Yes we do, I'll add memory management.